src/opening-period/time-span/TimeSpan.tsx

```diff
@@ -88,18 +90,23 @@ export default function TimeSpan({
               dataTest={`time-span-start-time-${groupIndex}-${index}`}
               registerFn={(ref): void =>
                 register(ref, {
-                  validate: (startTime): boolean =>
-                    validateTimeRange(
+                  validate: (startTime): boolean => {
+                    if (fullDay) {
```

Comment:
This condition could probably locate inside the validateTimeRange-function. Then the name of the function should be something else tough 🤔